source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRCarouselView.mm

```diff
@@ -0,0 +1,253 @@
+//
+//  ACRCarouselView.mm
+//  AdaptiveCards
+//
+//  Created by Abhishek Gupta on 25/09/24.
+//  Copyright © 2024 Microsoft. All rights reserved.
+//
+
+#import "ACOBaseCardElementPrivate.h"
+#import "ACOHostConfigPrivate.h"
+#import "ACRColumnView.h"
+#import "ACRContentHoldingUIView.h"
+#import "ACRImageProperties.h"
+#import "ACRTapGestureRecognizerFactory.h"
+#import "ACRUIImageView.h"
+#import "ACRView.h"
+#import "Enums.h"
+#import "Icon.h"
+#import "SharedAdaptiveCard.h"
+#import "UtiliOS.h"
+#import "ACRSVGImageView.h"
+#import "ACRSVGIconHoldingView.h"
+#import "CompoundButton.h"
+#import "ACRCompoundButtonRenderer.h"
+#import "ACRUILabel.h"
+#import "UtiliOS.h"
+#import "ACRCarouselViewRenderer.h"
+#import "Carousel.h"
+#import "ACRRenderer.h"
+#import "FlowLayout.h"
+#import "AreaGridLayout.h"
+#import "ACRFlowLayout.h"
+#import "ARCGridViewLayout.h"
+#import "ACRLayoutHelper.h"
+#import "CarouselPage.h"
+#import "ACRRendererPrivate.h"
+#import "ACRPageControl.h"
+#import "ACRCarouselPageView.h"
+#import "ACRCarouselPageContainerView.h"
+#import "ACRCarouselView.h"
+#import "ACRCarouselView.h"
+#import "ACRCarouselPageContainerView.h"
+
+@interface ACRCarouselView()
+
+@property NSInteger carouselPageViewIndex;
+@property ACRPageControl *pageControl;
+@property NSMutableArray<UIView *> *carouselPageViewList;
+@property std::shared_ptr<Carousel> carousel;
+
+@end
+
+@implementation ACRCarouselView
+
+-(instancetype) initWithViewGroup:(UIView<ACRIContentHoldingView> *)viewGroup
+                      rootView:(ACRView *)rootView
+                        inputs:(NSMutableArray *)inputs
+               baseCardElement:(ACOBaseCardElement *)acoElem
+                    hostConfig:(ACOHostConfig *)acoConfig
+{
+    self = [super initWithFrame:CGRectZero];
+    self.translatesAutoresizingMaskIntoConstraints = NO;
+    std::shared_ptr<BaseCardElement> elem = [acoElem element];
+    std::shared_ptr<Carousel> carousel = std::dynamic_pointer_cast<Carousel>(elem);
+    std::shared_ptr<HostConfig> config = [acoConfig getHostConfig];
+    self.carouselPageViewList = [[NSMutableArray alloc] init];
+    
+    self.carousel = carousel;
+    
+    
+    ACRColumnView *carouselViewContainer = [[ACRColumnView alloc] initWithStyle:(ACRContainerStyle)carousel->GetStyle()
+                                                        parentStyle:[viewGroup style]
+                                                         hostConfig:acoConfig
+                                                          superview:viewGroup];
+    
+    for(auto carouselPage: carousel->GetItems()) {
```

Comment:
We are following objective-C styled code formatting. 
for(..)
{
} 

if()
{
}
else
{
}

Please check in other places as well